app/uk/gov/hmrc/apisubscriptionfields/model/Model.scala

```diff
@@ -28,6 +29,16 @@ case class ApiVersion(value: String) extends AnyVal
 
 case class SubscriptionFieldsId(value: UUID) extends AnyVal
 
+object ValidationRuleType extends Enumeration {
+  type ValidationRuleType = Value
+
+  val REGEX = Value("REGEX")
+}
+
+case class ValidationRule(validationRuleType: ValidationRuleType, value: String)
+
+case class Validation(errorMessage: String, rules: Seq[ValidationRule])
```

Comment:
The Seq should probably be either a non empty list (cats NEL) or private constructor and an apply method that returns Option[Validation] that requires at least one rule.  If this class is to be used in a Form then we probably need to separate two types rather than allowing duff data into the db.